conf/gulp-tasks/extracttranslations.gulpfile.js

```diff
@@ -0,0 +1,43 @@
+const { src } = require('gulp');
+const { Writable } = require('stream');
+const path = require('path');
+const TranslationExtractor = require('../i18n/extract/translationextractor');
+
+module.exports = function extractTranslations () {
+  const extractor = new TranslationExtractor();
+
+  /**
+   * Extracts messages from a given source file.
+   * 
+   * @param {vinyl.File} file 
+   * @param {string} encoding 
+   * @param {Function} next 
+   */
+  function extractMessagesFromFile(file, encoding, next) {
```

Comment:
Do you think it would make sense for this to be the public method of the `TranslationExtractor` class? I was thinking it maybe nice to have the abstraction but I'm sure there are also reasons to have the function here!

Reply:
yeah I think we can move this logic in there, that seems more straightforward!